ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -27,24 +28,18 @@
         <include pattern="/etc/acs-commons/lists/report-parameter-components/jcr:content/list/default_\d+"/>
         <include pattern="/etc/acs-commons/notifications"/>
         <include pattern="/etc/acs-commons/notifications/jcr:content"/>
+        <include pattern="/etc/acs-commons/notifications/rep:policy(/.*)?"/>
         <include pattern="/etc/acs-commons/packagers"/>
         <include pattern="/etc/acs-commons/packagers/jcr:content"/>
         <include pattern="/etc/acs-commons/redirect-maps"/>
         <include pattern="/etc/acs-commons/redirect-maps/jcr:content"/>
+        <include pattern="/etc/acs-commons/redirect-maps/rep:policy(/.*)?" />
         <include pattern="/etc/acs-commons/exporters"/>
         <include pattern="/etc/acs-commons/exporters/jcr:content"/>
         <include pattern="/etc/acs-commons/qr-code"/>
-        <include pattern="/etc/acs-commons/qr-code/jcr:content"/>
-        <include
-            pattern="/etc/acs-commons/qr-code/jcr:content/clientlib-author"
-            mode="merge"/>
-        <exclude
-            pattern="/etc/acs-commons/qr-code/jcr:content/config"
-            mode="merge"/>
+        <include pattern="/etc/acs-commons/qr-code/jcr:content(/.*)?"/>
         <include pattern="/etc/acs-commons/instant-package"/>
-        <include pattern="/etc/acs-commons/instant-package/jcr:content"/>
-        <include pattern="/etc/acs-commons/instant-package/jcr:content/instant-package-image.png"/>
-        <include pattern="/etc/acs-commons/instant-package/jcr:content/config"/>
+        <include pattern="/etc/acs-commons/instant-package/jcr:content(/.*)?"/>
```

Comment:
If the tool saves some other nodes (from the AEM UI) under `jcr:content` will this new config now blow those away?

Reply:
yes, but AFAIK the tool stores the additional information in subpages, i.e. as siblings to jcr:content

Reply:
Should be fixed now.

Reply:
@kwin @HitmanInWis this theoretical situation can easily be made into an automated test by an oakpal scan with a forcedRoot of /etc/acs-commons/instant-package/jcr:content/config/oldNode and the basic/paths check.

Reply:
these forcedRoots probably just needs to be added to the acs-internal.json checklist.